noname.yaml

```diff
@@ -27,6 +27,7 @@ services:
     capabilities:
      - all
     net: host
+    pid: host
```

Comment:
Was this better than `CAP_SYS_BOOT`?

Reply:
All capabilities are passed above.. `pid:host` was needed to signal pid 1